packages/airnode-deployer/terraform/aws/main.tf

```diff
@@ -31,8 +32,79 @@ module "startCoordinator" {
   secrets_file       = var.secrets_file
 
   environment_variables = {
-    HTTP_GATEWAY_URL             = var.http_api_key == null ? null : "${module.httpGw[0].api_url}"
-    HTTP_SIGNED_DATA_GATEWAY_URL = var.http_signed_data_api_key == null ? null : "${module.httpSignedGw[0].api_url}"
+    AIRNODE_WALLET_PRIVATE_KEY = var.airnode_wallet_private_key
+  }
+
+  invoke_targets                 = [module.run.lambda_arn]
+  schedule_interval              = 1
+  reserved_concurrent_executions = var.disable_concurrency_reservation ? null : 1
+
+  depends_on = [module.run]
+}
+
+module "startCoordinatorHttpGw" {
+  source = "./modules/function"
+  count  = var.http_api_key != null && var.http_signed_data_api_key == null ? 1 : 0
```

Comment:
What is this count for? Also is it correct? (Currently, it's the same for all 4 options). Also, the `http_signed_data_api_key` seems strange here as this is for http gw.

Reply:
> Currently, it's the same for all 4 options

It's not the same for all 4 options, it's different for each option, that's the point. I have to provide a different module for each possibility:
* no gateways
* only HTTP gateway
* only signed data HTTP gateways
* both gateways

The `count` argument is always set to either 1 (there should be exactly 1 instance of this module) or 0 (there should be no instance of this module). I'm basically switching on and off copies of the same module based on what gateway(s) are enabled. And passing their output (this time without a condition) to its environment variables.

Reply:
Oh, sorry 🤦 . I see the difference now.